src/Processors/QtiV2/In/Constants.php

```diff
@@ -25,4 +25,6 @@ class Constants
     public static $unsupportedInteractions = [
         'drawingInteraction'
     ];
+    
+    public static $learnosityImagePath = 'https://assets.learnosity.com/organisations/';
```

Comment:
Can we rename this (for clarity and consistency) to `$baseLearnosityAssetsUrl`?

Reply:
Done